.github/workflows/helper_scripts/request_contributor_review.py

```diff
@@ -0,0 +1,142 @@
+        return True
+    else:
+        return False
+
+
+def get_pr_author(pr_number, github_token, verify_ssl):
+    pr_endpoint = f"https://api.github.com/repos/{REPO_OWNER}/{REPO_NAME}/pulls/{pr_number}"
+    headers = {"Authorization": "Bearer " + github_token} if github_token else {}
+
+    response = requests.get(pr_endpoint, headers=headers, verify=verify_ssl)
+
+    if response.status_code not in [200, 201]:
+        print(f"Failed in pulling PR {pr_number} data")
+        sys.exit(1)
+
+    pr_info = response.json()
+
+    return pr_info.get('user', {}).get('login', '').lower()
+
+
+def get_pr_modified_packs(pr_number, github_token, verify_ssl):
+    pr_endpoint = f"https://api.github.com/repos/{REPO_OWNER}/{REPO_NAME}/pulls/{pr_number}/files"
+
+    headers = {'Authorization': 'Bearer ' + github_token} if github_token else {}
+
+    response = requests.get(pr_endpoint, headers=headers, verify=verify_ssl)
+
+    if response.status_code not in [200, 201]:
+        print(f"Failed in pulling PR {pr_number} data")
+        sys.exit(1)
+
+    pr_changed_data = response.json()
+    pr_files = [f.get('filename') for f in pr_changed_data]
+    modified_packs = {Path(p).parts[1] for p in pr_files if p.startswith(PACKS_FOLDER) and len(Path(p).parts) > 1}
+
+    return modified_packs
+
+
+def request_review_from_user(reviewers, pr_number, github_token=None, verify_ssl=True):
+    review_endpoint = f"https://api.github.com/repos/{REPO_OWNER}/{REPO_NAME}/pulls/{pr_number}/requested_reviewers"
+    headers = {"Authorization": "Bearer " + github_token} if github_token else {}
+
+    reviewers_data = {
+        "reviewers": list(reviewers)
+    }
+
+    response = requests.post(review_endpoint, json=reviewers_data, headers=headers, verify=verify_ssl)
+
+    if response.status_code not in [200, 201]:
+        print(f"Failed requesting review on PR {pr_number}")
+        sys.exit(1)
+
+
+def check_pack_and_request_review(pr_number, github_token=None, verify_ssl=True):
+    modified_packs = get_pr_modified_packs(pr_number=pr_number, github_token=github_token, verify_ssl=verify_ssl)
+    pr_author = get_pr_author(pr_number=pr_number, github_token=github_token, verify_ssl=verify_ssl)
+    reviewers = set()
+
+    for pack in modified_packs:
+        pack_metadata_path = os.path.join(PACKS_FULL_PATH, pack, PACK_METADATA)
+
+        if os.path.exists(pack_metadata_path):
+            with open(pack_metadata_path, 'r') as pack_metadata_file:
+                pack_metadata = json.load(pack_metadata_file)
+
+            if pack_metadata.get('support') != XSOAR_SUPPORT and PACK_METADATA_GITHUB_USER_FIELD in pack_metadata \
+                    and pack_metadata[PACK_METADATA_GITHUB_USER_FIELD]:
+                pack_reviewers = pack_metadata[PACK_METADATA_GITHUB_USER_FIELD]
+                pack_reviewers = pack_reviewers if isinstance(pack_reviewers, list) else pack_reviewers.split(",")
+                github_users = [u.lower() for u in pack_reviewers]
```

Comment:
users are case insensitive?

Reply:
In gihtub api users are case insensitive for some reason.